test/utest/src/cuter_cerl_tests.erl

```diff
@@ -6,60 +6,43 @@
 -include("include/eunit_config.hrl").
 -include("include/cuter_macros.hrl").
 
--spec test() -> 'ok' | {'error', term()}.  %% This should be provided by EUnit
+-spec test() -> any().
 
--type descr() :: nonempty_string().
--type f_one() :: fun((term()) -> term()).
--type setup() :: {'setup', fun(() -> term()), f_one(), f_one()}.
--type ret_t() :: {descr(), setup()}.
-
-%% Load the modules
--spec just_load_test_() -> [ret_t()].
-just_load_test_() ->
-  Setup = fun(M) -> fun() -> setup(M) end end,
-  Cleanup = fun cleanup/1,
-  Inst = fun just_load/1,
-  [{"Load Module: " ++ atom_to_list(M), {setup, Setup(M), Cleanup, Inst}}
-   || M <- ?MODS_LIST].
-
-just_load({M, MDb}) ->
+-spec load_lists_module_test() -> any().
+load_lists_module_test() ->
   TagGen = fun() -> {?BRANCH_TAG_PREFIX, 42} end,
-  R = cuter_cerl:load(M, MDb, TagGen, false),
-  Ns = ets:lookup(MDb, name),
-  [{"successful loading", ?_assertEqual({ok, M}, R)},
-   {"retrieve module's name", ?_assertEqual([{name, M}], Ns)}].
+  {ok, Klists} = cuter_cerl:load(lists, TagGen, false),
+  R = cuter_cerl:destroy_kmodule(Klists),
+  ?_assertEqual(ok, R).
 
-%% Have the proper exports
--spec load_exports_test_() -> [ret_t()].
-load_exports_test_() ->
-  Setup = fun(M) -> fun() -> setup(M) end end,
-  Cleanup = fun cleanup/1,
-  Inst = fun load_exports/1,
-  [{"Validate exported funs: " ++ atom_to_list(M),
-    {setup, Setup(M), Cleanup, Inst}} || M <- ?MODS_LIST].
-  
-load_exports({M, MDb}) ->
+-spec exported_mfas_in_lists_module_test_() -> any().
```

Comment:
This function does not return `any()`; it returns a list of terms (of some type).